vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow/src/main/java/com/vaadin/flow/component/menubar/MenuBar.java

```diff
@@ -54,6 +54,7 @@
 @Tag("vaadin-menu-bar")
 @NpmPackage(value = "@vaadin/polymer-legacy-adapter", version = "23.3.0-alpha1")
 @JsModule("@vaadin/polymer-legacy-adapter/style-modules.js")
+@JsModule("./contextMenuConnector.js")
```

Comment:
Shouldn't there be `import './contextMenuConnector.js';` in `menubarConnector.js` because [menubar connector uses contextmenu connector.](https://github.com/vaadin/flow-components/blob/4a3963e07c25d180f5471d92c769de79f0a29762/vaadin-menu-bar-flow-parent/vaadin-menu-bar-flow/src/main/resources/META-INF/resources/frontend/menubarConnector.js#L63) Now it just assumes it to be available

Reply:
Good point, changed.